lib/tasks/limber.rake

```diff
@@ -155,6 +155,12 @@ namespace :limber do
       %w[WGS LCMB].each do |prefix|
         Limber::Helper::RequestTypeConstructor.new(prefix).build!
       end
+
+      Limber::Helper::RequestTypeConstructor.new(
+        'Duplex-Seq',
```

Comment:
So this constructor prefixes the request type with 'Limber' right?

Reply:
yes